docs/components/stories.md

```diff
@@ -40,8 +41,15 @@ exports.stories = [
 
 // A module generating dynamically editable knobs
 exports.knobs = require('./knobs');
+
+// Stories will be provided a clean instance of fetch-mock to allow mocking of http requests
+exports.fetchMock = fetchMock => {
+    fetchMock.mock('/api/data', { some: 'data' });
+}
```

Comment:
Would it make sense to add something to the example in the story configuration section below?

Reply:
Oops, it should be there instead of here!